code/ui/manager/src/components/mobile/navigation/MobileNavigation.tsx

```diff
@@ -86,4 +84,18 @@ const Button = styled.button(({ theme }) => ({
   fontSize: `${theme.typography.size.s2 - 1}px`,
   padding: '0 7px',
   fontWeight: theme.typography.weight.bold,
+  WebkitLineClamp: 1,
+
+  '> svg': {
+    width: 14,
+    height: 14,
+    flexShrink: 0,
+  },
+
+  '> p': {
+    display: '-webkit-box',
+    WebkitLineClamp: 1,
+    WebkitBoxOrient: 'vertical',
+    overflow: 'hidden',
+  },
```

Comment:
I think this is an anti-pattern, could we create a styled component instead?

Reply:
I fixed it @JReinhold